haskell_scotty/Client.hs

```diff
@@ -59,3 +64,13 @@ instance FromJSON Client where
       (attributes >>= (.: "phone")) <*>
       (attributes >>= (.: "created_at")) <*>
       (attributes >>= (.: "updated_at"))
+  parseJSON _ = error "unexpected non-object JSON"
+
+instance QueryResults Client where
+  convertResults fs vs =
+      Client $... constructorWrap (undefined :: Client) fs vs
+
+instance QueryParams Client where
+  renderParams
+    (Client _ name' address' city' state' zip' country' email' contact' phone' created_at' updated_at') =
```

Comment:
:+1: I'm glad I took the time to wrestle with `mysql-simple` and instancing `QueryResults` and `QueryParams`, because I learned a lot! But moving to `goundhog` seems like a good idea.